pynsee/utils/init_conn.py

```diff
@@ -134,8 +125,8 @@ def init_conn(
     # save config
     config = {
         "sirene_key": sirene_key,
-        "http_proxy": http_proxy,
-        "https_proxy": https_proxy,
+        "http_proxy": http_proxy if not http_proxy == "" else "DIRECT",
+        "https_proxy": https_proxy if not https_proxy == "" else "DIRECT",
```

Comment:
What's that?
Usually the default value is "" or None, what is the meaning of "DIRECT"?
Also the preferred syntax would be ``http_proxy or "DIRECT"``

Reply:
That something I thought I had removed 😳 !